app/client/ui/FormAPI.ts

```diff
@@ -139,15 +139,16 @@ export class FormAPIImpl extends BaseAPI implements FormAPI {
     });
   }
 
-  public async createRecord(options: CreateRecordOptions): Promise<void> {
+  public async createRecord(options: CreateRecordOptions): Promise<number> {
     const {tableId, colValues} = options;
-    return this.requestJson(
+    const response = await this.requestJson(
       this._docOrShareUrl(`/tables/${tableId}/records`, options),
       {
         method: "POST",
         body: JSON.stringify({ records: [{ fields: colValues }] }),
       }
     );
+    return response?.records?.[0]?.id;
```

Comment:
Is there any reason to worry that this may sometimes return `undefined`? If so, I'd update the return type to `number|undefined`, and when replacing `{{ID}}`, would prefer to be more explicit what we want in that case.

If there is really no realistic scenario of it though, then maybe OK not to worry about it, or add `|| 0` to satisfy the types.